openmrs/src/main/java/org/motechproject/openmrs/tasks/impl/OpenMRSActionProxyServiceImpl.java

```diff
@@ -133,6 +137,17 @@ public void updatePerson(String configName, String personUuid, String givenName,
     }
 
     @Override
+    public Visit createVisit(String configName, String patientUuid, DateTime startDatetime,
+                             DateTime endDatetime, String visitType, String locationName) {
+        Location location = getLocationByName(configName, locationName);
+        Patient patient = patientService.getPatientByUuid(configName, patientUuid);
+        VisitType type = new VisitType(visitType);
```

Comment:
I suppose that visitType is uuid, so why do you use this value as display in VisitType object ?

Additionally I suggest to rename variable to visitTypeUuid it should prevent mistakes